editor/src/components/canvas/canvas-strategies/reparent-strategy-helpers.ts

```diff
@@ -87,25 +85,19 @@ export function reparentStrategyForParent(
   )
 
   const newParentMetadata = MetadataUtils.findElementByElementPath(targetMetadata, newParent)
-  const parentProvidesBoundsForAbsoluteChildren =
-    newParentMetadata?.specialSizeMeasurements.providesBoundsForAbsoluteChildren ?? false
-
   const parentIsFlexLayout = MetadataUtils.isFlexLayoutedContainer(newParentMetadata)
-  const parentIsStoryboard = EP.isStoryboardPath(newParent)
 
   if (allDraggedElementsAbsolute) {
     if (parentIsFlexLayout) {
       return { strategy: 'ABSOLUTE_REPARENT_TO_FLEX', newParent: newParent }
-    }
-    if (parentProvidesBoundsForAbsoluteChildren || parentIsStoryboard) {
```

Comment:
this is definitely an unfinished part of the behavior. we shouldn't automatically reparent (without the user pressing _something_) into an element that doesn't provide bounds. 

we should leave at least a TODO comment here, but I wonder if this is a merge blocker.

Reply:
Yeah, this is what I meant when I said that the absolute conversion is too easy to trigger, as this is still the same case (right now) when dragging a non-absolute element to move it when it is already in a parent that doesn't provide bounds. This is what triggered the other spike around finding other valid targets for an interaction. I wouldn't say this is a merge blocker, but it's definitely something to keep at the forefront of our minds whilst we work on that spike.